web/package.json

```diff
@@ -41,9 +43,6 @@
     "@babel/core": "^7.0.0",
     "@babel/preset-env": "^7.11.0",
     "@babel/preset-react": "^7.10.4",
-    "@types/jasmine": "~3.5.0",
-    "@types/jasminewd2": "~2.0.3",
-    "@types/node": "^12.11.1",
```

Comment:
Why it's safe to remove these dependencies?

Reply:
I think they are removed when I did `npm install`. They are added back now!

Reply:
Why they've moved to the 'dependencies' section from the 'devDependencies' section then?

Reply:
I think I just made the same mistake...

Reply:
Instead of removing the duplicates here, can we remove the @type/xxx from above to keep the list sorted?